thrust/functional.h

```diff
@@ -58,9 +58,7 @@ template<typename Operation> struct binary_traits;
  *  };
  *  \endcode
  *
- *  \note unary_function is currently redundant with the C++ STL type
- *  \c std::unary_function. We reserve it here for potential additional
- *  functionality at a later date.
+ *  \note Inheriting from unary_function is optional if C+11 support is enabled.
```

Comment:
C+11 -> C++11

I think I would change the note to say something like:

"Because C++11 language support makes the functionality of \c unary_function obsolete, it's use is optional if C++11 language features are enabled."

Note that the `\c` makes Doxygen print the word in typewriter font.